api_client/python/timesketch_api_client/sketch.py

```diff
@@ -1240,13 +1240,21 @@ def search_by_label(self, label_name, as_pandas=False):
         return self.explore(
             query_dsl=json.dumps({'query': query}), as_pandas=as_pandas)
 
-    def add_event(self, message, timestamp, timestamp_desc):
+    def add_event(
+            self, message, timestamp, timestamp_desc, attributes=None,
+            tags=None):
         """Adds an event to the sketch specific timeline.
 
         Args:
             message: A string that will be used as the message string.
-            timestamp: Micro seconds since 1970-01-01 00:00:00.
+            timestamp: A string with the timestamp of the message.
```

Comment:
Is this a timestamp, or a datetime string?

Reply:
done